script/prerender/thread.ts

```diff
@@ -100,7 +100,7 @@ function makeSitemapIndexTask(services: AppOptions['services']) {
     );
     const items = await asyncPool(MAX_CONCURRENT_CONNECTIONS, books, async(book) => {
       const archiveBook = await getArchiveBook(services, book);
-      return getSitemapItemOptions(archiveBook, sitemapPath(book.params.book.slug));
+      return getSitemapItemOptions(archiveBook, `https://openstax.org${sitemapPath(book.params.book.slug)}`);
```

Comment:
this was apparently a discrepancy that was introduced when switching to the distributed pre-render, the old logic is here https://github.com/openstax/rex-web/blob/main/script/prerender/sitemap.ts#L60